src/card.css

```diff
@@ -0,0 +1,79 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+}
+
+.card__img {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+
+}
+
+.card__text {
+  margin: 0 0 4px;
```

Comment:
Would be more descriptive to just use margin-bottom. If you need to reset margins, do it in * {...} rule